src/server/protocol.ts

```diff
@@ -216,6 +217,27 @@ namespace ts.server.protocol {
         projectFileName?: string;
     }
 
+    /**
+     * Requests status information from the server
+     */
+    export interface StatusRequest extends Request {
+        command: CommandTypes.Status;
+    }
+
+    export interface StatusResponseBody {
+        /**
+         * the version of the currently-running server.

```

Comment:
Nit: We don't version the server independently of TypeScript as a whole (like we do with services) -- so I would say "The TypeScript version (`ts.version`)"

Reply:
changed.